types/dc/index.d.ts

```diff
@@ -63,34 +64,42 @@ declare namespace dc {
     }
 
     export interface Events {
-        trigger(fn: () => void, delay?: number): void;
+        trigger(closure: () => void, delay?: number): void;
```

Comment:
why the rename?

Reply:
In [Line 1231](https://github.com/dc-js/dc.js/blob/56df1dcd3f67201b750ef989aaa1c3e15632096d/dc.js#L1231),  apparently the function argument was renamed.